classwork/server-2.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+
+const app = express();
+app.use(express.json());
+
+let people = ["Jason"];
+
+app.get("/people", (request, response) => {
+  response.setHeader("Access-Control-Allow-Origin", "*");
+  response.json(people);
+});
+
+app.put("/people", (request, response) => {
+  people.push(...request.body);
+  response.json(people);
+});
+
+app.delete("/people", (request, response) => {
+res.send("DELETE Request Called");
```

Comment:
You may want to try to use VSCode's code formatting function (press CMD/CTRL + P, then enter "> format" at the prompt box) 

Are you deleting an element from the `people` array? Did you miss capturing the id from url?

Reply:
Many thanks for noticing - I've finished it now! Didn't know about CTRL + P + '> format', will learn more about this toll  now. Thank you!